CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+0.1.4 (2015-11-12)
```

Comment:
Related: The changes look backwards-incompatible. But we (and presumably everyone else) aren't actually using this, yet. So, I'm not too worried about sane versioning.